src/AppBundle/Utils/Slugger.php

```diff
@@ -27,6 +27,6 @@ class Slugger
      */
     public function slugify($string)
     {
-        return trim(preg_replace('/[^a-z0-9]+/', '-', strtolower(strip_tags($string))), '-');
+        return preg_replace('/\s+/', '-', mb_strtolower(trim(strip_tags($string))));
```

Comment:
you should force the encoding to UTF-8 for mb_strtolower. The internal encoding of mbstring might be configured to a different one